components/ollama/common/utils.mjs

```
@@ -0,0 +1,87 @@
+
+function parseOptions(options) {
+  if (!options) {
+    return;
+  }
+  return Object.fromEntries(
+    Object.entries(options)
+      .map(([
+        key,
+        value,
+      ]) => {
+        let parsedValue = isNaN(value)
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)